README.md

```diff
@@ -1,5 +1,7 @@
-# OpenMATERIAL
-A new ASAM standard with the target to develop a 3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
+# ASAM OpenMATERIAL 3D
+ASAM OpenMATERIAL 3D is a new ASAM standard with the target to develop a 3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
```

Comment:
I would remove "is a new ASAM standard". This sentence gets out-dated quite fast ;).
And let's call it "3D asset exchange format" instead of 3D model.

Reply:
Agreed, I changed it accordingly.